test/functional/wallet_resendwallettransactions.py

```diff
@@ -33,7 +33,7 @@ def run_test(self):
         time.sleep(1.1)
 
         # Can take a few seconds due to transaction trickling
-        wait_until(lambda: node.p2p.tx_invs_received[txid] >= 1, lock=mininode_lock)
+        wait_until(lambda: node.p2p.tx_invs_received[txid] >= 1, lock=p2p_lock)
```

Comment:
I've been waiting for someone to touch these lines so I can recommend using this function I made 😂  
Avoid using the test_framework.util `wait_until` and also avoid using the `p2p` property.
```suggestion
        node.p2ps[0].wait_for_broadcast([txid])
```
This requires that you don't convert `txid` to an int (i.e. get rid of int(..., 16)) which I think is a good thing.

Reply:
These all seem like fine suggestions, but out of scope for this renaming PR.